mod_elasticsearch2.erl

```diff
@@ -194,6 +358,13 @@ terminate(_Reason, _State) ->
 
 %%%%%%%%%%%%%%%%%%%%%%%%%% private functions %%%%%%%%%%%%%%%%%%%%%%%%%%
 
+take(0, L, Acc) ->
```

Comment:
Zit dit niet al in OTP? https://www.erlang.org/docs/18/man/lists.html#split-2

Reply:
Erlang `lists:split/2` can't handle lists shorter than N.

```
(zotonic001@PoToi)1> lists:split(10, [a]).
** exception error: bad argument
     in function  lists:split/2
        called as lists:split(10,[a])
```